### PR TITLE
.ci: Fix and clean runtime installation

### LINKED ISSUE #499: .ci: Fix and clean runtime installation for Jenkins CI

### PR DESCRIPTION
**Pull request by sboeuf**
This commit cleans up the installation of Docker service for Clear
Containers. It also fixes the dockerd command line having some issues
if using -D flag on latest Docker version 17.06.

Fixes #499

### CODE REVIEW

File: .ci/install_runtime.sh [hunk 1]
```diff
@@ -53,22 +53,22 @@ echo "Enabling global logging for runtime in file ${runtime_config_path}"
 sudo sed -i -e 's/^#\(\[runtime\]\|global_log_path =\)/\1/g' "${runtime_config_path}"
 
 echo "Add runtime as a new/default Docker runtime. Docker version \"$(docker --version)\" could change according to Semaphore CI updates."
-docker_options="-D --add-runtime cc30=/usr/local/bin/cc-runtime --default-runtime=cc30"
+docker_options="--add-runtime cc30=/usr/local/bin/cc-runtime --default-runtime=cc30"
```
Comment (jodh-intel) [inline .ci/install_runtime.sh]: What issues are we seeing with `-D`? This is a useful option so we don't really want to have to remove it imho.

Comment (sboeuf) [reply .ci/install_runtime.sh]: I don't know why I was not able to restart the docker service in the past when this was enabled but apparently I have this working now. Nevermind...

**Comment by sboeuf** [comment]
@chavafg please take a look

**Comment by chavafg** [comment]
lgtm

**Pull request merged**